### ISSUE TITLE
Calyptos does not work when disconnected from the internet

### ISSUE THREAD

**Issue by viglesiasce** [eucalyptus/calyptos#33]
In cases where the internet is not accessible calyptos fails at various stages:
- Unable to install chef-dk
- Unable to pull dependencies using berks
- Unable to pull packages from Riak repos

I will continue to edit this as I find more issues

**Comment by shaon** [comment]
I was wondering if we could build an all-in-one package for calyptos and release with every Eucalyptus release, so that users can get the package and deploy eucalyptus to places/datacenters where there is limited or restricted internet facility available.

**Comment by viglesiasce** [comment]
Yep @shaon I'm thinking something like that as well. I was thinking of shipping a full VM image that has all the dependencies including package mirrors with our dependencies.
